nicsdru_unity_theme.libraries.yml

```diff
@@ -4,6 +4,7 @@ unity-global-styling:
   js:
     js/empty-tags-script.js: { minified: true }
     js/ext-win-script.js: { minified: true }
+    js/unity-menu-script.js: { minified: true }
```

Comment:
fairly minor point, can the order in which scripts are listed be changed so that unity-menu-script is first & the empty-tags-script is last